tlh.md

```diff
@@ -0,0 +1,47 @@
+996.ICU
+===
+
+## nuq 'oH 996
+jaj naQ Qap, ICU loS.
+
+996 vum qej Qap schedule (9 am-9 pm, 6 jaj net Hogh) 'e' Soq vI'Iprup Qat. 
```

Comment:
In Klingon Languages, there is no am or pm concept, instead there is a stardate timing system.